keps/sig-node/1769-memory-manager/README.md

```diff
@@ -618,9 +618,9 @@ _This section must be completed when targeting beta graduation to a release._
 
 
 ###### What specific metrics should inform a rollback?
-
   The pod may fail with the admission error because the kubelet can not provide all resources aligned from the same NUMA node. 
   You can see the error message under the pod events.
+  "memory_manager_assignment_errors_total".
```

Comment:
This seems a bit abrupt. Can we add a sentence here to explain the data reflected by this metric and how it can be used.

Reply:
Sure. I added a detailed explanation at the comment itself, but you're right, it deserves a more details here as well.

Reply:
Added a bit more details